tools/biolatency-kp.bt

```diff
@@ -8,6 +8,11 @@
  * Copyright 2018 Netflix, Inc.
  * Licensed under the Apache License, Version 2.0 (the "License")
  *
+ * This version of the tool uses kprobes to attach to kernel events.
```

Comment:
Should we bother keeping this in the main tools folder or move it to "old" or the user-tools repo?

Reply:
It has been in the main repo from the very beginning as a way to show how kprobes can be used (to achieve a similar thing to tracepoints). I'm not sure if it's worth the work to keep it there but it's out of the scope of this PR so I'd just leave it for now.